docs/parse_source.py

```diff
@@ -11,6 +11,56 @@
 if not os.path.exists(".db"):
     os.makedirs(".db")
 
+replace_table = {
+    # standard basic types
+    "uint8_t": "int",
+    "uint16_t": "int",
+    "uint32_t": "int",
+    "uint64_t": "int",
```

Comment:
Best to probably move it to separate file completely, so no one will be tempted to just use it here directly

Reply:
Oh maybe, it just seemed silly that you have to edit it in three places when 2 need to be the same.